### PR TITLE
Fixes vertx-sql-common#21: Implement the isolation level API

### PR DESCRIPTION
**Pull request by pmlopes**
Fixes vert-x3/vertx-sql-common#21

The client implementation of the API

### CODE REVIEW

File: src/main/java/io/vertx/ext/jdbc/impl/JDBCConnectionImpl.java [hunk 1]
```diff
@@ -128,4 +129,57 @@ public SQLConnection rollback(Handler<AsyncResult<Void>> handler) {
     new JDBCRollback(vertx, conn, context).execute(handler);
     return this;
   }
+
+  @Override
+  public SQLConnection setTransactionIsolation(TransactionIsolation isolation) {
+    try {
+      switch (isolation) {
+        case READ_COMMITTED:
+          conn.setTransactionIsolation(Connection.TRANSACTION_READ_COMMITTED);
```
Comment (beatdigit) [inline src/main/java/io/vertx/ext/jdbc/impl/JDBCConnectionImpl.java]: Thanks for this!

Just one question.
I suppose setTransactionIsolation is very similar with setAutoCommit, both of them need to talk with the actual server. So this call could be blocked by network operation.
setAutoCommit use executeBlocking to run the underlying code, my question is why this one doesn't need executeBlocking? Am I missing something here?

Thanks!

**Comment by pmlopes** [comment]
It depends on the driver, but you're right it is probably a safer bet to make this method async.

**Comment by vietj** [comment]
👍 beware : this should be merged using the new WorkerExecutor usage that is now in vertx-jdbc-client

**Comment by pmlopes** [comment]
I've fixed all the conflicts and updated the code for the new executor

**Comment by vietj** [comment]
looks ok

**Pull request merged**